<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextOpenClass</class>
 <widget class="QMainWindow" name="TextOpenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文本阅读器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TextOpen.qrc">
    <normaloff>:/images/logo.ico</normaloff>:/images/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="TextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_8"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>跳转</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>查找</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>编码</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_6"/>
   <addaction name="menu_5"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/skip.png</normaloff>:/images/skip.png</iconset>
   </property>
   <property name="text">
    <string>跳转页面</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="text">
    <string>查找内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/findUp.png</normaloff>:/images/findUp.png</iconset>
   </property>
   <property name="text">
    <string>查找上一个</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/findDown.png</normaloff>:/images/findDown.png</iconset>
   </property>
   <property name="text">
    <string>查找下一个</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
   <property name="shortcut">
    <string>Shift+F</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="TextOpen.qrc">
     <normaloff>:/images/saveAs.png</normaloff>:/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ANSI</string>
   </property>
  </action>
  <action name="action_12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>TextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TextOpen.qrc"/>
 </resources>
 <connections/>
</ui>
